pkg/bootkube/status.go

```diff
@@ -122,11 +138,18 @@ func (s *statusController) PodStatus() (map[string]v1.PodPhase, error) {
 			return nil, err
 		}
 		if !exists {
-			status[watchedPod] = doesNotExist
+			status[watchedPod] = nil
```

Comment:
nit: `delete(status, watchedPod)` ?

Reply:
then the loop above gets more complicated. Not worth the change.

Reply:
ack.